### PR TITLE
Query capabilities support

### PR DESCRIPTION
**Pull request by jtary**
Add command for checking if a particular feature is available on the
process.

connect to strongloop-internal/scrum-nodeops#453

### CODE REVIEW

**Comment by sam-github** [comment]
LGTM

**Comment by jtary** [comment]
@sam-github PTAL

File: server/models/service-process.js [hunk 1]
```diff
@@ -333,6 +333,34 @@ module.exports = function extendServiceProcess(ServiceProcess) {
...
+    this._appCommand({
+        cmd: 'query-capabilities',
+        target: this.pid,
+        feature: feature,
+      }, callback);
+  }
+  ServiceProcess.prototype.queryCapabilities = queryCapabilities;
+
+  /**
+   * Check to see if all capabilities that are supported.
```
Comment (sam-github) [inline server/models/service-process.js]: stray "if"

Comment (sam-github) [inline server/models/service-process.js]: Caller's of the API would probably thank you for documenting the format of what's returned (not that our API is a paragon of thorough documentation). Maybe just inject some example output.

**Comment by sam-github** [comment]
still looks good

**Pull request merged**